Source/WebCore/html/HTMLAttachmentElement.h

```diff
@@ -101,12 +110,19 @@ class HTMLAttachmentElement final : public HTMLElement {
     bool childShouldCreateRenderer(const Node&) const final;
 #endif
 
+    enum class Implementation { Legacy, Modern, LegacyUnderModern };
```

Comment:
`enum class Implementation : uint8_t`

Reply:
`LegacyUnderModern` is a confusing name – maybe `IconOnly`?

Reply:
"LegacyUnderModern" because it's really a legacy <attachment> that is always a child of, and is controlled by, a modern <attachment>.
It could be more than an icon: Thumbnail, placeholder...
Other suggestions?
(Note that eventually the whole legacy code should be removed, once we've fully transitioned.)